analyzer/test/regression/uva/test-29-safe.cpp

```diff
@@ -0,0 +1,23 @@
+#include <memory>
+
+struct MyStruct {
+  int _aa;
+  int _bb;
+  MyStruct() noexcept
+  {
+    _aa = 5;
+    _bb = 7;
+  }
+};
+
+int main(int argc, char *argv[])
+{
+  // This test gives a false positive because the ikos AR
+  // does not assure that the body is executed at least once.
+  // That could probably be fixed.
```

Comment:
Ah, yes, that's another problem we have seen before.
I think we talked about unrolling the loop but then that messes up with debug information and warning. It's also a bit delicate to do during the control flow graph construction.